### ISSUE TITLE
Automatically generate BIDS compulsory agnostic files and update them if they already exist

### ISSUE THREAD

**Issue by vinferrer** [physiopy/phys2bids#242]
comes from #185. When a heuristic file is provided and data is bidsdifyed agnostic files (README, participants.tsv and dataset_description.json) should be generated
## Detailed Description
When a heuristic file is provided and data is bidsdifyed agnostic files should be generated:

- [ ] dataset_description.json
       Fields
       - Name (Required)
       - BIDSVersion (Required)
       - DatasetType (Recommended)
       - License (Recommended)

- [ ] Participants.tsv
       Fields
       - participant_id (Required)
       - age (Recommended)
       - sex (Recommended)
       - handedness (Recommended)

- [ ] README

## Context / Motivation
comply BIDS 1.4 standard

## Possible Implementation
Create one function per file where we check if the file exists in the dataset dir, in case it doesn't generate it and cover at least the required fields, preferably also the  recommended  and add them to the pipeline.

**Comment by smoia** [comment]
Thank you @vinferrer !

Indeed, until now we weren't creating metadata, relying on other tools (DICOM based) to do so. But if physiological files should be shared on their own, we need to create them.

**Comment by eurunuela** [comment]
Thanks @vinferrer ! I guess it automatically closed when the #244 merged.

**Comment by vinferrer** [comment]
Yeah don't worry it happens, I was looking for the issue and didn't find it so i supposed it was close

**Comment by vinferrer** [comment]
We should close this issue

**Comment by eurunuela** [comment]
I'd wait for #256.
